swan-accpy/Dockerfile

```diff
@@ -7,6 +7,7 @@ ARG NB_UID="1000"
 ARG BUILD_TAG=daily
 ENV VERSION_DOCKER_IMAGE=$BUILD_TAG
 ENV ACCPY_PATH="/opt/acc-py"
+ENV JAVA_HOME="/usr"
```

Comment:
Can you explain why we needed this again? The Spark connector does not find Java unless this is defined? Is it because it relies on JAVA_HOME and can't look for java in the path?

Also, why not defining JAVA_HOME to /usr/lib/jvm/java-11-openjdk-11.0.24.0.8-2.el9.x86_64?

Reply:
Yes, JAVA_HOME needs to be explicitly defined in order to spark to find it. I tried to define JAVA_HOME using `readlink -f $(which java)` but isn't something that can be done in the Dockerfile. I avoided the path you mention for being too volatile, so I went for the safest option, which would be using '/usr', considering `/bin/java` (appended by the spark connector) would be pointing to the same java and it is more static/reliable

Reply:
I would remove this for now.